cmd/beacon-chain/flags/base.go

```diff
@@ -16,11 +16,12 @@ var (
 		Usage: "A MEV builder relay string http endpoint, this wil be used to interact MEV builder network using API defined in: https://ethereum.github.io/builder-specs/#/Builder",
 		Value: "",
 	}
-	// HTTPWeb3ProviderFlag provides an HTTP access endpoint to an ETH 1.0 RPC.
-	HTTPWeb3ProviderFlag = &cli.StringFlag{
-		Name:  "http-web3provider",
-		Usage: "A mainchain web3 provider string http endpoint. Can contain auth header as well in the format --http-web3provider=\"https://goerli.infura.io/v3/xxxx,Basic xxx\" for project secret (base64 encoded) and --http-web3provider=\"https://goerli.infura.io/v3/xxxx,Bearer xxx\" for jwt use",
-		Value: "http://localhost:8545",
+	// ExecutionEngineEndpoint provides an HTTP access endpoint to connect to an execution client on the execution layer
+	ExecutionEngineEndpoint = &cli.StringFlag{
+		Name:    "execution-endpoint",
+		Usage:   "An execution client http endpoint. Can contain auth header as well in the format",
+		Value:   "http://localhost:8551",
+		Aliases: []string{"http-web3provider"},
```

Comment:
I'm tempted to add this alias, so it's not a complete breaker, anyone has an opinion here?